Gemfile.next.lock

```diff
@@ -123,8 +124,9 @@ GEM
       thor (~> 1.0)
     builder (3.2.4)
     byebug (11.1.3)
-    capybara (3.35.3)
+    capybara (3.36.0)
```

Comment:
@neutrino are all these updates needed? I see updates for capybara, selenium, omniauth-oauth2, rubocop, seem unrelated to objective of this PR